db/schema.rb

```diff
@@ -23,6 +23,10 @@
     t.datetime "updated_at", null: false
   end
 
+  add_index "blockings", ["blocked_id"], name: "index_blockings_on_blocked_id", using: :btree
+  add_index "blockings", ["blocker_id", "blocked_id"], name: "index_blockings_on_blocker_id_and_blocked_id", unique: true, using: :btree
+  add_index "blockings", ["blocker_id"], name: "index_blockings_on_blocker_id", using: :btree
+
   create_table "follows", force: :cascade do |t|
     t.string   "follower_type"
     t.integer  "follower_id"
```

Comment:
Any thoughts about what caused this? My guess is that it was generated with rake db:migrate.

My experience is that this does not hurt performance.